nfc_magic/scenes/nfc_magic_scene_gen4_get_info.c

```diff
@@ -1,11 +1,14 @@
 #include "../nfc_magic_app_i.h"
+//TODO: INCAPSULATE?
+#include "gui/scene_manager.h"
+#include "protocols/gen4/gen4_poller_i.h"
```

Comment:
"scene_manager.h" is already included in #include "../nfc_magic_app_i.h"

Reply:
Don't include private API. There should be function in gen4_poller.h
`const Gen4* gen4_poller_get_data(Gen4Poller* instance);`